docs/class_diagram/class_diagram.md

```diff
@@ -1,15 +1,14 @@
 ```mermaid
 classDiagram
-    Game --> "2" Player : has
-    Game --> "1" Deck : has
-    Game --> "5" DiscardPile : has
-    Player --> "1" Hand : has
-    Player --> "5" Expedition : has
-    Expedition --> "*" Card : contains
-    AbstractPile o-- "*" Card : contains
+    Game --o "2" Player : has
+    Game --o "1" Deck : has
+    Game --o "5" DiscardPile : has
+    Player --o "1" Hand : has
+    Player --o "5" Expedition : has
+    Expedition --* "*" Card : contains
```

Comment:
Ez itt fordítva van.
A has/contains címkék meg nem kellenek, pont ezt jelölik a nyílhegyek. Olyankor kell címke, ha valamiből mondjuk több van, de más role-ban. Például ha a currentPlayer egy Player referencia lenne, akkor a Game és a Player közt lenne egy aggregáció meg egy asszociáció kapcsolat is, akkor fel kéne címkézni hogy melyik melyik.
De nálad gondolom a currentPlayer egy index. Mondjuk jó lenne jelölni a tagváltozók típusait is.